request.go

```diff
@@ -146,20 +146,43 @@ func getArgsTypeList(args []interface{}) (string, error) {
 	return types, nil
 }
 
+type Request struct {
+	Params      interface{}
+	Attachments map[string]string
+}
+
+// Pls just use NewRequest to get a Request.
```

Comment:
change the comment to `NewRequest create a new request`

Reply:
ok